### DISCUSSION TITLE
This is an amazing project

### DISCUSSION THREAD

**Discussion by Vinlic** [remotion-dev/remotion#2957] [General]
This is an amazing project, and I just completed my latest open-source project today: [WebVideoCreator](https://github.com/Vinlic/WebVideoCreator). It seems to share some similarities with remotion. Perhaps we can discuss some interesting aspects together :D.

**Comment by JonnyBurger** [comment]
Thanks a lot, also find your project very impressive!

Also fascinated by what you did to solve the non-realtime animation problem.

Happy to discuss any aspect you like! You can post here or reach me on Discord under @jonnyburger

**Comment by Vinlic** [comment]
@JonnyBurger For deterministic rendering of the video element, has remotion ever attempted [VideoDecoder](https://developer.mozilla.org/en-US/docs/Web/API/VideoDecoder) decoding? It seems to have a lower cost.

**Reply by JonnyBurger** [reply]
Not yet, but we developed an OffthreadVideo component (https://www.remotion.dev/docs/offthreadvideo), which essentially does the same thing, manually decoding it on the native side in a Rust thread
